python/src/main/python/streaming-llm/base_requirements.txt

```diff
@@ -1,5 +1,6 @@
 apache-beam[gcp]==2.68.0
 jsonschema==4.25.1 # TODO: remove when correct version could come from beam
+jsonschema-specifications==2025.4.1 # TODO: remove when correct version could come from beam
```

Comment:
@liferoad @Amar3tto is one of you following up with the Beam-side change?

Reply:
Also, do we have an example job that is failing without this which could be linked in this PR description?